plugins/hls-stan-plugin/test/testdata/test.hs

```diff
@@ -0,0 +1,4 @@
+orderPair x y
+    | x < y = 1
+    | x > y = 2
+    | otherwise = 3
```

Comment:
The test directory contains some compiler outputs, such as .o and .hi.

Reply:
Fixed. Shouldn't they be in gitignore? I can add them in this or separate PR.

Reply:
I think we (most of HLS contributors) rarely compile files directly so they aren't in .gitignore (the file contains stack's and cabal's working directories only)